src/Cache/RedisFactory.php

```diff
@@ -50,7 +50,7 @@ private function normalizeServer(string $server): array
         $parsedServer = parse_url(trim($server));
         if (! is_array($parsedServer)) {
             throw new InvalidArgumentException(sprintf(
-                'Provided server "%s" is not a valid URL with format schema://[[username]:password@]host:port',
+                'Provided server "%s" is not a valid URL with format schema://[[username]:password@]host:port/database',
```

Comment:
Since the database is not mandatory, perhaps it should be better as `[/database]`, like it's done with the credentials.

```suggestion
                'Provided server "%s" is not a valid URL with format schema://[[username]:password@]host:port[/database]',
```